ext/oboe_metal/src/oboe.i

```diff
@@ -0,0 +1,29 @@
+%module oboe
```

Comment:
I think this `oboe.i` file is downloaded during gem build and probably should not be checked in? If so, maybe should add it to .gitignore

Reply:
Removed.